packages/fxa-settings/README.md

```diff
@@ -43,6 +43,14 @@ const LogoImage = () => (
 const LogoImage = () => <div class="logo" role="img" aria-label="logo"></div>;
 ```
 
+## Storybook
+
+This project uses [Storybook](https://storybook.js.org/) to show each screen without requiring a full stack.
+
+_You will eventually be able to view the built Storybook at <http://mozilla.github.io/fxa/settings>._
```

Comment:
When this is merged, will you comment on #4823 with a link to this line so we'll be sure to update this piece of the readme when it's ready?

BTW, side note, I'm thinking once CSS ADR part 2 + the GQL ADR is merged we can update this README with info about the stack as well. Thanks for maintaining this file.